stdlib/gc.mli

```diff
@@ -149,7 +149,30 @@ type control =
         out variations in its workload. This is an integer between
         1 and 50.
         Default: 1. @since 4.03.0 *)
-}
+
+    custom_major_ratio : int;
+    (** A percentage relative to the major heap size. This parameter
+        controls the major GC speed as a function of memory allocated
+        outside the heap and retained by custom values located in the
+        major heap. The GC will speed up as needed to (try to) limit
+        the amount of floating garbage to this much memory. The
+        default value keeps the out-of-heap floating garbage about the
+        same size as the in-heap overhead.
+        Default: 44. @since 4.08.0 *)
```

Comment:
Don't you want to mention here that these new settings only apply to custom values allocated using the new API?

Reply:
Right.